benchmark/plot.sh

```diff
@@ -0,0 +1,50 @@
+# pip install openrlbenchmark==0.2.1a5
+PR_662_TAG=v0.4.7-55-g110e672
+PR_662_NAME=PR-662
+
+python -m openrlbenchmark.rlops_multi_metrics \
```

Comment:
I am very confused by how these work :D They don't seem to be using the same `benchmark/benchmark.py` right? Can you somewhere document what that command exactly does, otherwise it looks hard to maintain for anybody else.

Reply:
Nice catch! The documentation is https://github.com/openrlbenchmark/openrlbenchmark#get-started. Should I just put the link to the documentation at the top of `plot.sh`?

Reply:
ok sounds good